keyboards/handwired/justageng/office60/readme.md

```diff
@@ -0,0 +1,127 @@
+
+To check that dfu-util can detect dfu-capable device, run the following in command prompt:
+```
+dfu-util -l
+```
+to list available devices. There should be *three* options with the name "1eaf:0003", corresponding to the main, legacy, and protected version provided by the bootloader, shown below. 
+
+```
+C:\Users\Jia Geng>dfu-util -l
+dfu-util 0.11
+
+Copyright 2005-2009 Weston Schmidt, Harald Welte and OpenMoko Inc.
+Copyright 2010-2021 Tormod Volden and Stefan Schmidt
+This program is Free Software and has ABSOLUTELY NO WARRANTY
+Please report bugs to http://sourceforge.net/p/dfu-util/tickets/
+
+Found DFU: [1eaf:0003] ver=0201, devnum=36, cfg=1, intf=0, path="1-1", alt=2, name="STM32duino bootloader v1.0  Upload to Flash 0x8002000", serial="LLM 003"
+Found DFU: [1eaf:0003] ver=0201, devnum=36, cfg=1, intf=0, path="1-1", alt=1, name="STM32duino bootloader v1.0  Upload to Flash 0x8005000", serial="LLM 003"
+Found DFU: [1eaf:0003] ver=0201, devnum=36, cfg=1, intf=0, path="1-1", alt=0, name="STM32duino bootloader v1.0  ERROR. Upload to RAM not supported.", serial="LLM 003"
+```
+
+
+We will flash bootloader into main version of the device, with alternate=2.
+
+```
+dfu-util -d 1eaf:0003 -a 2 -D /path/to/qmk/firmware/file 
+```
+where ``/path/to/qmk/firmware/file`` is the path to the Office60 keyboard firmware ``.bin`` file (e.g. ``/c/msys64/home/Jia_Geng/qmk_firmware/handwired_justageng_office60_default.bin``).
+
+If it works, press the reset button on the blue pill, or reconnect the USB. Running ``lsusb`` should show a device with the vid/pid "0FF1:CE60". This will show up as a keyboard in device manager. 
+
+Note: If you get an error like "LIBUSB_ERROR_IO" when running the last dfu-util command, then your bluepill is not compatible. Example error message shown below:
+```
+C:\Users\Jia Geng>dfu-util -d 1eaf:0003 -a 2 -D ..\..\msys64\home\Jia_Geng\qmk_firmware\handwired_justageng_office60_default.bin
+dfu-util 0.11
+
+Copyright 2005-2009 Weston Schmidt, Harald Welte and OpenMoko Inc.
+Copyright 2010-2021 Tormod Volden and Stefan Schmidt
+This program is Free Software and has ABSOLUTELY NO WARRANTY
+Please report bugs to http://sourceforge.net/p/dfu-util/tickets/
+
+Failed to retrieve language identifiers
+Failed to retrieve language identifiers
+Opening DFU capable USB device...
+Device ID 1eaf:0003
+Device DFU version 0110
+Claiming USB DFU Interface...
+Setting Alternate Interface #2 ...
+Cannot set alternate interface: LIBUSB_ERROR_IO
+```
+
+You probably got a lemon from the manufacturer, and I do not know how to fix this apart from buying a new bluepill and praying that it does not come with the incompatible version. This sour lemon bluepill also gives a strange output for dfu-util list when the BOOT1 jumper is disconnected (if BOOT1 jumper is connected, the --list output is identical to the compatible bluepill - qmk firmware will not be flashed successfully even if command prompt says its successful):
```

Comment:
Colloquialisms like "a lemon" may not be understood or translated very well, please change to "a defective unit" or similar.